### PR TITLE
PM8550b eUSB2 repeater

### PR DESCRIPTION
**Pull request by superna9999**
Add status for PM8550b eUSB2 repeater

### CODE REVIEW

File: pmic.yaml [hunk 1]
```diff
@@ -78,3 +78,6 @@ bms:
   name: BMS
 clkdiv:
   name: clkdiv
+eusb2repeat:
+  intop: True
```
Comment (lumag) [inline pmic.yaml:82]: Do we want that to have on the front page?

File: _pmic/pm8550b.md [hunk 2]
```diff
@@ -24,5 +24,7 @@ pmic-rtc:
 pmic-tempalarm: 6.3
 pmic-typecpd:
 pmic-wled:
+pmic-eusb2repeat: WIP
+pmic-eusb2repeat-comment: repeater driver support was merged in 6.3
```
Comment (lumag) [inline _pmic/pm8550b.md]: Then it is 6.3, not WIP

File: pmic.yaml (hunk 1 above)
Comment (superna9999) [reply pmic.yaml:82]: Yes, but I can remove it

File: _pmic/pm8550b.md (hunk 2 above)
Comment (superna9999) [reply _pmic/pm8550b.md]: Ack

**Pull request merged**